UITests/LoginManagerTests.swift

```diff
@@ -628,7 +628,7 @@ class LoginManagerTests: KIFTestCase {
         passwordCell = list.cellForRow(at: IndexPath(row: 2, section: 0)) as! LoginTableViewCell
         XCTAssertEqual(passwordCell.descriptionLabel.text, "changedpassword")
 
-        tester().tapView(withAccessibilityLabel: "Logins")
+        tester().tapView(withAccessibilityLabel: "Logins & Passwords")
```

Comment:
@garvankeeley Hi! I tried to set and id for this string instead of using label here but could not fine where to do the modification. Could you please point me to the correct place to do it? as long as that does not imply big changes...Thanks!

Reply:
There is `Strings.LoginsAndPasswordsTitle`, however, the accessibility label should have stayed the same as it is set here:
https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Frontend/Settings/AppSettingsOptions.swift#L757

Reply:
Edit: sorry my comment is wrong, that link is for the Settings button accessibility label. Maybe that should be updated too, not sure.

Reply:
Yeah, that's for the Logins button in Settings. Not sure if its name should be changed...although the string changed since we were using the id tests were not affected (only UI Tests needs the modification) ...I will try to find the other button to add its id...let me know if you can remember any other place where it could be added...thanks!